### ISSUE TITLE
Add `setName` to Card Model

### ISSUE THREAD

**Issue by staghouse** [mtgjson/mtgjson#776]
I believe it would be beneficial to include `setName` to the card model (Card, Card (Token), and Card (Deck)). Right now there is only `setCode` which is more meta-data like than it is useful.

In a particular use case example, when using the SQLite database one has to execute on the cards table, find the set code and then execute on the sets table to match and locate the set name - then do more work to store that upwards.

In short, this is helpful data for any developer in my opinion. Any other thoughts?

**Comment by ZeldaZach** [comment]
I'm not sold on this change tbh. SetCode can be mapped to SetName, if required. It also bloats the files _that_ much more, without giving any tangible benefits imo. If I'm wrong here, please do let me know!

For the SQLite, we could always add that data point in, if it'll help our users. I just don't see the use for MTGJSON itself.

**Comment by staghouse** [comment]
Adding it to the SQL/ite databases is definitely a win since the size of the file is less an issue

**Comment by staghouse** [comment]
Who would normally take on this work?
